source/_integrations/python_script.markdown

```diff
@@ -37,15 +37,15 @@ It is not possible to use Python imports with this integration. If you want to d
 This is a simplified example that does no real work.
 It is created as a first step, to help with:
 
-- Demonstrate how to setup the script
+- Demonstrating how to setup the script
 - How to process the input data
 - How to log the script activity
 - How to troubleshoot / manually call the script.
 
-Start by enabling the Python script and create the first script.
+Start by enabling the Python_script integration and create the first script.
```

Comment:
This change is incorrect, this is normal language, not code.

```suggestion
Start by enabling the Python script integration and create the first script.
```

Reply:
frenck, thanks for looking at this, the integration is actually titled "Python Scripts"
https://www.home-assistant.io/integrations/python_script/
I propose:
... the Python **S**cript**s** integration ...
(Changes bolded just for this conversation)

Does that work for you?

Reply:
Adjusted the suggestion